### PR TITLE
Astro 1/24 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Review by kwcanuck — APPROVED** [review]

**Comment by chomipi88** [comment]
Additions:

- Ingress to Data Plane clusters (for Airflow UI/API access) is now locked down to just the Control Plane IPs. Roll out to all Data Plane clusters will take place over the next 1-2 weeks.

https://app.zenhub.com/workspaces/astro---all-teams-5ffc69dc8d499a00116e0db3/issues/gh/astronomer/astro/7980

- Support for custom tagging of AWS Data Plane clusters via support. Visibility into applied tags via Clusters page in Cloud UI

https://app.zenhub.com/workspaces/astro---all-teams-5ffc69dc8d499a00116e0db3/issues/gh/astronomer/astro/6448

**Review by chomipi88 — COMMENTED** [review]
See comment

File: astro/release-notes.md [hunk 1]
```diff
@@ -23,6 +23,17 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.9.0 ([Release notes](cli/release-notes.md))
 
+## January 24, 2023
+
+### Additional improvements
+
+- You can now create new clusters in France Central for Bring Your Own Cloud installations of Astro on Azure.
```
Comment (jwitz) [inline astro/release-notes.md:30]:
```suggestion
- Ingress to clusters in the data plane is now limited only to IPs from the control plane. This change will arrive to all clusters over the next few weeks.
- You can now request custom tags for your clusters through [Astronomer support](https://cloud.astronomer.io/support). You can view your cluster tags in the Cloud UI by going to **Clusters**, selecting a cluster, and clicking **Details**. See [View clusters](view-clusters.md).
- You can now create new clusters in France Central for Bring Your Own Cloud installations of Astro on Azure.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.9.0 ([Release notes](cli/release-notes.md))
 
+## January 24, 2023
+
+### New Workspace homepage
```
Comment (jwitz) [inline astro/release-notes.md:28]:
```suggestion
### New Workspace home page 
```

File: astro/airflow-alerts.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ Use your existing Amazon SES instance to send Airflow alerts by email.
     - Click **Download Credentials** or copy them and store them in a safe place.
     - Click **Close Window**.
 
-7. In the Cloud UI, select a Workspace and then select a Deployment.
+7. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/airflow-alerts.md:107]:
```suggestion
7. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/api-keys.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ A new API key framework to unlock advanced automation workflows will soon be ava
 
 ## Create an API key
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/api-keys.md:35]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/api-keys.md [hunk 5]
```diff
@@ -78,7 +78,7 @@ For more information and examples, see [Automate code deploys with CI/CD](ci-cd.
 
 If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/api-keys.md:81]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/configure-deployment-resources.md [hunk 6]
```diff
@@ -48,7 +48,7 @@ If you experience delays in task execution, which you can track with the Gantt C
 
 #### Edit scheduler settings 
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:51]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/airflow-alerts.md (hunk 3 above)
Comment (kwcanuck) [reply astro/airflow-alerts.md:107]: Alternatively, you can use _select_ as recommended in our secondary style guide:

image

File: astro/configure-deployment-resources.md [hunk 7]
```diff
@@ -58,7 +58,7 @@ If you experience delays in task execution, which you can track with the Gantt C
 
 ## Update a Deployment name and description
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:61]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/configure-deployment-resources.md [hunk 8]
```diff
@@ -68,7 +68,7 @@ If you experience delays in task execution, which you can track with the Gantt C
 
 Alert emails assigned to a Deployment are used by Astronomer support to notify recipients in the case of an issue with the Deployment. This can include a problem with your scheduler or workers. Automated email alerts are coming soon.
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:71]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/configure-deployment-resources.md [hunk 9]
```diff
@@ -88,7 +88,7 @@ To transfer a Deployment from one Workspace to another, the Workspaces must be i
 
 Only the users who are members of the target Workspace can access the Deployment after it is transferred. To transfer a Deployment, you must be a Workspace Admin or Editor in both the original Workspace and the target Workspace.
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:91]:
```suggestion
1. In the Cloud UI, select a Workspace, select **Deployments**, and then select a Deployment.
```

File: astro/configure-worker-queues.md [hunk 10]
```diff
@@ -88,7 +88,7 @@ If you prefer, you can also run the `astro deployment worker-queue create` comma
 
 :::
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/configure-worker-queues.md:91]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/configure-worker-queues.md [hunk 11]
```diff
@@ -140,7 +140,7 @@ You can update worker queue configurations at any time. The worker queue name ca
 
 If you need to change the worker type of an existing worker queue, Astronomer recommends making the change at a time when it will not affect production pipelines. After you've changed a worker type, Astronomer recommends waiting a minimum of five minutes before pushing new code to your Deployment.
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/configure-worker-queues.md:143]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/environment-variables.md [hunk 12]
```diff
@@ -45,7 +45,7 @@ If you prefer to work with the Astro CLI, you can create and update environment
 
 After you set an environment variable key, only the environment variable value can be modified. You can modify environment variables that are set as secret. However, the variable value is never shown. When you modify a secret environment variable, you'll be prompted to enter a new value.
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/environment-variables.md:48]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

File: astro/release-notes.md [hunk 13]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.9.0 ([Release notes](cli/release-notes.md))
 
+## January 24, 2023
+
+### New Workspace home page
```
Comment (kwcanuck) [inline astro/release-notes.md:28]:
```suggestion
### New Workspace Home page 
```

File: astro/release-notes.md [hunk 14]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.9.0 ([Release notes](cli/release-notes.md))
 
+## January 24, 2023
+
+### New Workspace home page 
+
+When you open a Workspace in the Cloud UI, you now enter the Workspace **Home** page. On this page, you can:
+
+- Check the status of your Deployments at a glance.
```
Comment (kwcanuck) [inline astro/release-notes.md:32]:
```suggestion
- Check the status of your Deployments.
```

File: astro/release-notes.md [hunk 15]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- Jump back in to the Deployments and Cloud IDE projects you most recently accessed.
```
Comment (kwcanuck) [inline astro/release-notes.md:33]:
```suggestion
- Quickly access your most recently viewed Deployments and Cloud IDE projects. 
```

File: astro/release-notes.md [hunk 16]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
 
+## January 24, 2023
+
+### New Workspace home page 
+
+When you open a Workspace in the Cloud UI, you now enter the Workspace **Home** page. On this page, you can:
+
+- Check the status of your Deployments at a glance.
+- Jump back in to the Deployments and Cloud IDE projects you most recently accessed. 
+- View release notes for all primary Astro products.
```
Comment (kwcanuck) [inline astro/release-notes.md:34]:
```suggestion
- View release notes for all Astro products.
```

File: astro/release-notes.md [hunk 17]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+[Workspace home page in the Cloud UI](/img/release-notes/workspace-home.png)
+
+The previous default Workspace page is now accessible in **Deployments**.
```
Comment (kwcanuck) [inline astro/release-notes.md:38]:
```suggestion

```

File: astro/release-notes.md [hunk 18]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+### Additional improvements
+
+- Ingress to clusters in the data plane is now limited only to IPs from the control plane. This change will arrive to all clusters over the next few weeks.
```
Comment (kwcanuck) [inline astro/release-notes.md:42]:
```suggestion
- Ingress to clusters in the data plane is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
```

File: astro/release-notes.md [hunk 19]
```diff
@@ -23,6 +23,31 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- You can now request custom tags for your clusters through [Astronomer support](https://cloud.astronomer.io/support). You can view your cluster tags in the Cloud UI by going to **Clusters**, selecting a cluster, and clicking **Details**. See [View clusters](view-clusters.md).
```
Comment (kwcanuck) [inline astro/release-notes.md:43]:
```suggestion
- You can now request custom tags for your clusters by submitting a support request to [Astronomer support](https://cloud.astronomer.io/support). You can view your cluster tags in the Cloud UI by selecting **Clusters**, selecting a cluster, and then clicking the **Details** tab. See [View clusters](view-clusters.md).
```

File: astro/upgrade-runtime.md [hunk 20]
```diff
@@ -99,7 +99,7 @@ Once you upgrade to a Deployment on Astro to a new version of Astro Runtime, you
 
 ## Confirm your upgrade on Astro
 
-1. In the Cloud UI, select a Workspace and then select a Deployment.
+1. In the Cloud UI, select a Workspace, go to **Deployments**, and then select a Deployment.
```
Comment (kwcanuck) [inline astro/upgrade-runtime.md:102]:
```suggestion
1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
```

**Review by kwcanuck — APPROVED** [review]

File: astro/release-notes.md [hunk 21]
```diff
@@ -23,6 +23,30 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+When you select a Workspace in the Cloud UI, the **Home** page now appears first. On this page, you can:
+
+- Check the status of your Deployments.
+- Quickly access your most recently viewed Deployments and Cloud IDE projects. 
+- View release notes for all Astro products.
+
+[Workspace home page in the Cloud UI](/img/release-notes/workspace-home.png)
+
+
```
Comment (jwitz) [inline astro/release-notes.md:38]:
```suggestion
```

File: astro/release-notes.md [hunk 22]
```diff
@@ -23,6 +23,30 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+### New Workspace Home page 
+
+When you select a Workspace in the Cloud UI, the **Home** page now appears first. On this page, you can:
+
+- Check the status of your Deployments.
+- Quickly access your most recently viewed Deployments and Cloud IDE projects. 
+- View release notes for all Astro products.
+
+[Workspace home page in the Cloud UI](/img/release-notes/workspace-home.png)
```
Comment (jwitz) [inline astro/release-notes.md:36]:
```suggestion
![Workspace home page in the Cloud UI](/img/release-notes/workspace-home.png)
```

**Pull request merged**